rs/messaging/src/routing/stream_handler.rs

```diff
@@ -849,7 +849,8 @@ impl StreamHandlerImpl {
                                 return Some((reason, msg));
                             }
                             RequestOrResponse::Response(response) => {
-                                // Critical error, responses should always be inducted successfully.
+                                // Critical error, guaranteed responses (and best-effort responses passing
+                                // the exceptions above) should always be inducted successfully.
```

Comment:
This should be updated to say something like "Responses should always be inducted successfully (or silently dropped, in the case of duplicate best-effort responses). But never produce an error."

Reply:
Done.